dataplane-webserver/src/routes/secrets.rs

```diff
@@ -251,20 +256,23 @@ fn is_valid_id(s: &str) -> bool {
         ("secret_name", example="readonly-role", description = "Secret name"),
         ("password", example="hef8wergWF9uh39hf93h", description = "New password")
     ),
+    request_body = PasswordString,
     responses(
         (status = 200,
             description = "Password successfully changed."),
         (status = 403,
             description = "Not authorized for query"),
     )
 )]
-#[patch("/secrets/{secret_name}/passwords/{password}")]
+#[patch("/secrets/{secret_name}/passwords")]
```

Comment:
Can you please change the path to
```
/api/v1/orgs/{org_id}/instances/{instance_id}/secrets/{secret_name}
```

Reply:
Input validation contains at least:

admin role in the org they are trying to modify

only allow the secret names "app-role", "readonly-role", and "superuser-role"

validate the password is at least 16 characters long